frontend/src/modules/modals/MultiStepWizard.tsx

```diff
@@ -827,20 +768,19 @@ const MultiStepWizard = (): JSX.Element => {
               </Grid>
             </Grid>
             <Grid item className={classes.marginContent}>
-              {multiParams.map((param: Data) => {
+              {multiParams.map((param: InternalState) => {
                 if (param.isSetvalEnabled) {
-                  if (param.setValue !== param.setValueActual) {
+                  if (param.ValueDraft !== param.ValueCurrent) {
                     return (
-                      <Typography variant="subtitle1">{`Change ${param.label} to ${param.setValue} ${param.units}?`}</Typography>
+                      <Typography variant="subtitle1">{`Change ${param.label} to ${param.ValueDraft} ${param.units}?`}</Typography>
```

Comment:
It would be great if we could display the old value, too: e.g. Change FiO2 from 21% to 50%?